libgadget/partmanager.h

```diff
@@ -29,13 +29,13 @@ struct particle_data
         unsigned int IsGarbage            :1; /* True for a garbage particle. readonly: Use slots_mark_garbage to mark this.*/
         unsigned int Evaluated            :1; /* True if already query already ran in treewalk */
         unsigned int DensityIterationDone :1; /* True if the density-like iterations already finished; */
-        unsigned int IsNewParticle        :1; /* True if the particle is created this step; used in SFR */
```

Comment:
Changing this to something like spare_8 in place may be an easier changeset

Reply:
Since there are now 8 spares, I thought of just shrinking struct particle_data by a word, what do you think?

Reply:
8 bits is a byte, not a word. Leaving some spares is always a good idea.

Reply:
I took them out - it is very easy to add them back if we need them. It isn't like struct particle_data is external ABI!